containerized.mk

```diff
@@ -4,6 +4,26 @@ DOCKER_IMAGE_DIR=${GOPATH}/src/${PROJECT_ROOT}
 
 DOCKER_SWARMKIT_DELVE_PORT ?= 2345
 
+# HOW TO USE CONTAINERIZED BUILDS:
+#
+# There are lots of flags that we've added to containerized builds of swarmkit,
+# mostly to support the various swarmkit dev's workflows. These can be kind of
```

Comment:
nit: `devs' workflows`?

Reply:
my high school english teachers would be very disappointed in me